Automattic/wp-calypso
# Adds undeclared dependencies

#### Changes proposed in this Pull Request

Adds undeclared dependencies to package.json.

These dependencies were been flagged by `@yarnpkg/doctor`:

```
➤ YN0000: │ /Users/sergio/src/automattic/wp-calypso/bin/build-metadata.js:2:17: Undeclared dependency on request
➤ YN0000: │ /Users/sergio/src/automattic/wp-calypso/bin/generate-devdocs-search-index.js:13:14: Undeclared dependency on lunr
➤ YN0000: │ /Users/sergio/src/automattic/wp-calypso/bin/generate-gutenboarding-design-thumbnails.js:27:24: Undeclared dependency on capture-website
➤ YN0000: │ /Users/sergio/src/automattic/wp-calypso/bin/generate-gutenboarding-design-thumbnails.js:29:15: Undeclared dependency on sharp
➤ YN0000: │ /Users/sergio/src/automattic/wp-calypso/test/test/helpers/use-nock/index.js:4:1: Undeclared dependency on debug
```

#### Testing instructions

* Run `node ./bin/build-metadata.js` and check it still works. (It generates a file in`./client/components/phone-input/data.js`, and after reformatting it with prettier, it shows lots of changes, but unrelated to having a explicit dependency on `request`).

* Delete `./client/server/devdocs/search-index.js`, run `yarn build-devdocs:search-index` and verify the file is re-created without changes.

* Follow the instructions inside `./bin/generate-gutenboarding-design-thumbnails.js`. Then run `node ./bin/generate-gutenboarding-design-thumbnails.js` and check it generates the files without errors

* Run `DEBUG="calypso:*" yarn test-client client/extensions/woocommerce/state/data-layer/request/test/index.js` and check you get a few `calypso:test:use-nock` lines in the output


#### Questions

* What is `./client/components/phone-input/data.js`? Should we re-generate it? cc @umurkontaci
